Sources/ChatUI/Appearance/ImageAppearanceScheme.ChatUI.swift

```diff
@@ -0,0 +1,164 @@
+/// Provides images from ``ChatSymbols`` with the specific color scheme.
+///
+/// The initializer allows to set up the ``ChatSymbols`` for both light and dark mode.
+///
+/// Example usage:
+/// ```
+/// // To get ``ChatSymbols.send`` for dark mode
+/// ImageAsset.send(.dark)
+/// // To get ``ChatSybmols.camera`` for light mode
+/// ImageAsset.camera(.light)
+/// ```
+
+import Foundation
+import SwiftUI
+
+public struct ImageAsset {
+    
+    private let darkSymbol: ChatSymbols
+    private let lightSymbol: ChatSymbols
```

Comment:
```suggestion
    private let universalSymbol: ChatSymbols
```
The default values of the images should be the light mode images always.
IMO, using `universalSymbol` in lieu of ``lightSymbol`` seems good. Please let me know your thought

Reply:
I think lightSymbol, darkSymbol, is clearer and simpler as it reflects app light and dark modes,